src/formatDate.js

```diff
@@ -50,7 +50,53 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let year = '';
+  let month = '';
+  let day = '';
+
+  const oldFormat = date.split(fromFormat[3]);
+
+  fromFormat.forEach((el, i) => {
```

Comment:
smth like `datePart` would be more fitting
```suggestion
  fromFormat.forEach((datePart, i) => {
```